### DISCUSSION TITLE
Gorhom's React Native Side Menu

### DISCUSSION THREAD

**Discussion by OfficialDarkComet** [gorhom/react-native-bottom-sheet#1060] [Ideas]
**First off, I love your work, and wanted to thank you. I especially love your Portal library which I use throughout my project.** Your Bottom Sheet library is also fantastic, so much so, that I have incorporated a bottom sheets into my designs.

Unfortunately, my side menu, design wise great, except for it is not draggable and your amazing bottom sheet component dwarfs my side menu due to the lack of draggability. 

After a couple of attempts, I instead, came up with the idea of modifying your library by more or less, turning it 90° degrees: vertical to horizontal, height to width, x to y, etc... I didn't want to re-write the library, so I did some "substitutions". The idea was to maintain what your did with the bottom sheet library and apply that to a "side sheet" or side menu container.

My plan was to use this new "Side Menu" in my project and then give you to working library, as nearly all of it is your code and then it would be up to you if you'd like to publish it under your Github.

**Now, my request. Things didn't go as smoothly as I wanting and I've run into some hiccups. Is there any chance you can look at what I did and tell me where I messed up?**

Things are not scaling correctly, though I'm still working on it to see if I can fix it. But if you can look at it and help me fix it, then I can test it and incorporated it into my project, and you are free to upload it under your Github and don't need to even mention me (as what I've contributed is just to idea and nothing compared to what you have done).

Thank you for everything you have contributed and the amazing libraries you've created.

\- Logan (_aka DarkComet_)

**Comment by phoenix-error** [comment]
Yes I would love this too
